src/chunker/layout.js

```diff
@@ -385,17 +440,25 @@ class Layout {
 				break;
 			}
 
+			node = node.parentNode;
 		}
 		return breakNode;
 	}
 
-	createBreakToken(overflow, rendered, source) {
+	createOverflow(overflow, rendered, source) {
 		let container = overflow.startContainer;
 		let offset = overflow.startOffset;
 		let node, renderedNode, parent, index, temp;
 
 		if (isElement(container)) {
-			temp = child(container, offset);
+			if (container.nodeName == "INPUT") {
+				temp = container;
+			} else {
+				// Offset can be incorrect in tables spanning multiple pages.
+				// To reproduce, fill a specification composite "prepared by" field
```

Comment:
Not clear what this means? Is this something we should add a new spec to test for?

Reply:
I don't remember that one - will have to look at it more.

Reply:
The comment was added by a co-worker. I tried removing the `|| child(container, 0)`, which they also added and it did result in 4 test failures, but in every case there was no noticable actual difference so I believe the issue they were addressing is already covered by other changes and tests.

Removing the modification they made.